pkg/manifests/types.go

```diff
@@ -19,392 +19,444 @@ import (
+	// `enforcedSampleLimit`. 
+	// Administrators can use this setting to keep the overall number of 
+	// samples under control.
+	// The default value is `0`, which means that no limit is set.
 	EnforcedSampleLimit *uint64 `json:"enforcedSampleLimit,omitempty"`
-	// EnforcedTargetLimit defines a global limit on the number of scraped targets. This overrides
-	// any TargetLimit set per ServiceMonitor or/and PodMonitor. It is meant to be used by admins to
-	// enforce the TargetLimit to keep the overall number of targets under the desired limit. Note
-	// that if TargetLimit is lower, that value will be taken instead, except if either value is
-	// zero, in which case the non-zero value will be used. If both values are zero, no limit is
-	// enforced.
-	// default: 0
+	// Specifies a global limit on the number of scraped targets. 
+	// This setting overrides the `TargetLimit` value set in any user-defined 
+	// `ServiceMonitor` or `PodMonitor` object if the value is greater than 
+	// `enforcedSampleLimit`. 
+	// Administrators can use this setting to keep the overall number of 
+	// targets under control.
+	// The default value is `0`.
 	EnforcedTargetLimit *uint64 `json:"enforcedTargetLimit,omitempty"`
-	// ExternalLabels defines labels to be added to any time series or alerts when communicating
-	// with external systems (federation, remote storage, Alertmanager).
-	// default: nil
+	// Defines labels to be added to any time series or alerts when 
+	// communicating with external systems such as federation, remote storage, 
+	// and Alertmanager.
+	// By default, no labels are added.
 	ExternalLabels map[string]string `json:"externalLabels,omitempty"`
-	// LogLevel defines the log level for Prometheus.
-	// Possible values are: error, warn, info, debug.
-	// default: info
+	// Defines the log level setting for Prometheus.
+	// The possible values are `error`, `warn`, `info`, and `debug`.
+	// The default setting is `info`.
 	LogLevel string `json:"logLevel,omitempty"`
-	// NodeSelector defines which Nodes the Pods are scheduled on.
+	// Defines the nodes on which the pods are scheduled.
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
-	// QueryLogFile specifies the file to which PromQL queries are logged. Suports both just a filename
-	// in which case they will be saved to an emptyDir volume at /var/log/prometheus, if a full path is
-	// given an emptyDir volume will be mounted at that location. Relative paths not supported,
-	// also not supported writing to linux std streams.
-	// default: ""
+	// Specifies the file to which PromQL queries are logged. 
+	// This setting can be either a filename, in which
+	// case the queries are saved to an `emptyDir` volume 
+	// at `/var/log/prometheus`, or a full path to a location where
+	// an `emptyDir` volume will be mounted and the queries saved. 
+	// Relative paths are not supported, nor is writing to 
+	// Linux `std` text streams.
```

Comment:
same as PrometheusK8sConfig

Reply:
@JoaoBraveCoding What are the allowable settings for `std` text streams?

Reply:
Simon mentioned them in this comment https://github.com/openshift/cluster-monitoring-operator/pull/1773/files#r989952365

> We support /dev/stderr, /dev/stdout or /dev/null. Any other /dev/xxx path is invalid.

Reply:
Revised as follows:

```
// Writing to `/dev/stderr`, `/dev/stdout` or `/dev/null` is supported, but 
// writing to any other `/dev/` path is not supported.
```